### PR TITLE
doc: add section about status bar segments

### PR DESCRIPTION
**Pull request by Delapouite**
Hi

I recently asked myself "How many status bar segments are provided by Dance?". After a small visit in the code, I think it would be a nice addition to have the answer available directly in the readme.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,17 @@ being:
 - `i + 1` replaces `1,1,1,1,1` into `1,2,3,4,5`, assuming that each selection is
   on a different digit.
 
+### Status bar
+
+Dance provides several status bar segments (left-aligned) exposing info similar
+to Kakoune's default mode-line:
```
Comment (71) [inline README.md:156]: I would also add that most of them are hidden by default, and only shown contextually. You can either write this, or explain it individually for each segment.

File: README.md [hunk 2]
```diff
@@ -150,6 +150,17 @@ being:
...
+
+- current mode: click to switch to another mode
+- macro recording status: click to stop recording
+- current count prefix: click to reset to 0
+- current register: click to unset
+- dance error: click to copy the last one
```
Comment (71) [inline README.md:162]: nit:
```suggestion
- dance error: click to copy the full description of the last error
```

(Or something else that implies that the full error is copied, not just what's shown on the status bar, which is truncated)

**Review by 71 — CHANGES_REQUESTED** [review]
Thanks! A few comments:

**Comment by Delapouite** [comment]
Thanks for the feedback. I've adjusted the text accordingly.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Dance
+j# Dance
```
Comment (71) [inline README.md:1]: Typo?

**Review by 71 — APPROVED** [review]
Thanks!

**Pull request merged**